src/front-door/azext_front_door/_params.py

```diff
@@ -16,11 +18,18 @@
     validate_frontend_endpoints, validate_backend_pool, MatchConditionAction)
 
 
+class RouteType(str, Enum):
+    forward = "Forward"
+    redirect = "Redirect"
```

Comment:
Is this enum not present in your SDK?

Reply:
No, its not present in SDK. SDK implementation is polymorphism based and there is no route type defined.